src/components/Card/Card.tsx

```diff
@@ -8,33 +8,12 @@ export interface CardProps {
   displayName: string;
   icon?: keyof typeof APIConfigurationIcons;
   description: string;
-  onClick: () => void;
 }
 
-export const Card: FunctionComponent<PropsWithChildren<CardProps>> = ({apiId, displayName, icon, description, onClick, children}) => {
+export const Card: FunctionComponent<PropsWithChildren<CardProps>> = ({apiId, displayName, icon, description, children}) => {
   const TitleIcon = icon ? APIConfigurationIcons[icon] : APIConfigurationIcons.GenericIcon;
 
-  const onCardClick = (event: MouseEvent) => {
-    // By-pass click if we actually clicked on a button (or it's children)
-    const clickedAButton = event.target instanceof Element && event.target.closest('button');
-    if (!clickedAButton) {
-      onClick();
-    }
-  }
-
-  const onKeyDown = (event: KeyboardEvent<HTMLElement>) => {
-    if (event.target !== event.currentTarget) {
-      return;
-    }
-
-    if (["Enter", "Space"].includes(event.code)) {
```

Comment:
Are `Enter` and `Space` keys still working to navigate?

Reply:
`Enter` still works but not `Space`. My other concern with using `Link` is that we're losing `const clickedAButton = event.target instanceof Element && event.target.closest('button');` that would prevent a redirect from clicking a button inside the component. I don't think this currently causes any issues, but could be annoying if we're trying to add buttons to these Cards in the future.

Reply:
There is actually a button! 

![image](https://github.com/RedHatInsights/api-documentation-frontend/assets/3845764/b8f6e865-fe3e-412e-b712-bdbae822fa8d)

That's the reason of that part

Reply:
@josejulio I missed that! Oops

So maybe we use the first approach for now, so that we can keep that logic not to interfere with the button press? I'm open to suggestions.

Reply:
I think (not 100% sure) that you can use a Link or a plain button with the `a` component and still keep the `onClick` and the default right click as @tahmidefaz suggested. 

I'm unsure why I added the `Space` into the list of keys :thinking:, but maybe we can remove it if it's not the default behavior of links.

Reply:
I'm talking about this setup:
![image](https://github.com/RedHatInsights/api-documentation-frontend/assets/3845764/4bc59b57-098f-482a-86c2-8e1563ae6c55)
![image](https://github.com/RedHatInsights/api-documentation-frontend/assets/3845764/e8cea0a3-2a0a-4ea3-b02e-0415737be283)

https://patternfly-react.surge.sh/components/button

Reply:
The Link component handles
- Ctrl+Click
- Right Click -> Open in New Tab
- Keyboard 'Enter' ('Space' is not default to click a link)

On a standard click, I'm preventing the default event so that we can ensure we are still preventing the redirect on a button click.

Reply:
I could also remove the `onClick` method and just call navigate from inside the Card component, that way we don't have to pass the `to` prop twice.

https://github.com/RedHatInsights/api-documentation-frontend/pull/215/commits/46fab6ffbb4eb08fdbffa7d75858bc45d2555f28